### PR TITLE
Default values for automatically added airports #829

### PR DESCRIPTION
**Pull request by YashGovekar**
Fix for Issue #829 and minor change of Updater redirecting to Login instead of Admin.

### CODE REVIEW

File: app/Database/seeds/settings.yml [hunk 1]
```diff
@@ -26,6 +26,20 @@
...
+  options:
+  type: int
+  description: If an airport's Ground Handling Cost Cost isn't added, set this value by default
+- key: general.default_jetA_fuel_cost
+  name: 'Default Jet A Fuel Cost'
+  group: general
+  value: 0.7
+  options:
+  type: text
+  description: If an airport's Jet A Fuel Cost isn't added, set this value by default
```
Comment (nabeelio) [inline app/Database/seeds/settings.yml:42]: Add these after the `acars` section, call the group `airports`

File: app/Database/seeds/settings.yml [hunk 2]
```diff
@@ -26,6 +26,20 @@
...
   type: boolean
   description: If an airport isn't added, try to look it up when adding schedules
+- key: general.default_ground_handling_cost
+  name: 'Default Ground Handling Cost'
+  group: general
+  value: 250
+  options:
+  type: int
+  description: If an airport's Ground Handling Cost Cost isn't added, set this value by default
+- key: general.default_jetA_fuel_cost
```
Comment (nabeelio) [inline app/Database/seeds/settings.yml:36]: Rename to: `airports.default_jet_a_fuel_cost`, fully lowercase

File: app/Database/seeds/settings.yml [hunk 3]
```diff
@@ -26,6 +26,20 @@
   options:
   type: boolean
   description: If an airport isn't added, try to look it up when adding schedules
+- key: general.default_ground_handling_cost
```
Comment (nabeelio) [inline app/Database/seeds/settings.yml:29]: `airports.default_ground_handling_cost`

File: app/Services/ImportExport/AirportImporter.php [hunk 4]
```diff
@@ -45,8 +45,16 @@ public function import(array $row, $index): bool
         $row['id'] = $row['icao'];
         $row['hub'] = get_truth_state($row['hub']);
 
+        if ($row['ground_handling_cost'] === null && $row['ground_handling_cost'] !== 0) {
```
Comment (nabeelio) [inline app/Services/ImportExport/AirportImporter.php:48]: Make sure these rows get casted to floats

File: app/Services/Importers/AirportImporter.php [hunk 5]
```diff
@@ -21,19 +21,34 @@ public function run($start = 0)
...
             'hub',
+            'ground_handling_cost',
+            'fuel_jeta_cost',
         ];
 
         $count = 0;
         $rows = $this->db->readRows($this->table, $this->idField, $start, $fields);
         foreach ($rows as $row) {
+            $ground_handling_cost = $row->ground_handling_cost;
+            $fuel_jetA_cost = $row->fuel_jeta_cost;
```
Comment (nabeelio) [inline app/Services/Importers/AirportImporter.php:32]: You need to cast it here, otherwise your checks below (`!==`) won't work since they're strings

Comment (YashGovekar) [reply app/Services/Importers/AirportImporter.php]: Is it correct now?

**Pull request merged**